downstream/assemblies/hub/assembly-faq.adoc

```diff
@@ -1,38 +1,95 @@
 [id="assembly-faq"]
-== Frequently asked questions about certification
+= Frequently asked questions about {CertifiedCon}
 
-=== Why certify Ansible collections?
+The following is a list of Frequently Asked Questions for the {PlatformName} Certification Program. 
+If you have any questions regarding the items below, please email ansiblepartners@redhat.com.
 
-The Ansible Certification Program is a shared statement of support for Certified Ansible collections between Red Hat and the ecosystem partner. An end customer, experiencing trouble with Ansible and certified partner content, can open a support ticket with Red Hat and expect the ticket will be resolved by Red Hat and the ecosystem partner. As a partner, publishing Collections to the Automation Hub gives your end customer organizations the power to manage how trusted automation content is used in the customer environment.
 
-=== How does the certification process work?
+== Why certify Ansible collections?
 
-Please visit Ansible.com/partners and review the Ansible certification policy guide to learn more about the certification process.
+The Ansible certification program enables a shared statement of support for {CertifiedCon} between Red Hat and the ecosystem partner. 
+An end customer, experiencing trouble with Ansible and certified partner content, can open a support ticket, for example, a request for information, or a problem with Red Hat, and expect the ticket to be resolved by Red Hat and the ecosystem partner. 
 
-=== Why should I publish my collection to Automation Hub?
+Red Hat offers go-to-market benefits for Certified Partners to grow market awareness, demand generation and collaborative selling.
 
-Collections present in Automation Hub are meant to be for joint customers of Red Hat and partners. Customers will need a subscription to download these collections and the collections present on Automation Hub will be marked as Certified. Certified Collection means that Red Hat and partners have a strategic relationship in place and are ready to support joint customers.
+Red Hat {CertifiedName} are distributed through {HubNameMain} (subscription required), a centralized repository for jointly supported Ansible Content. 
+As a certified partner, publishing collections to {HubNameMain} provides end customers the power to manage how trusted automation content is used in their production environment with a well-known support life cycle.
 
-=== What value do my customers get after downloading a collection from Automation Hub?
+For more information about getting started with certifying a solution, see https://www.ansible.com/partners.
```

Comment:
Suggest you change this to a link and include Alt text: 

link:https://www.ansible.com/partners[Red Hat Partner Connect].